libraries/chain/wasm_interface.cpp

```diff
@@ -987,6 +987,13 @@ class authorization_api : public context_aware_api {
       return context.is_account( account );
    }
 
+   void get_code_version( const account_name& account, int64_t& last_code_update_time, fc::sha256& code_hash ) {
```

Comment:
intrinsics with multiple output parameters have very strange semantics when the pointers alias.  I would strongly recommend either using a packed struct or splitting this into two intrinsics.